src/py_semantic_taxonomy/adapters/routers/router.py

```diff
@@ -244,9 +275,23 @@ async def concept_scheme_delete(
 # Relationship
 
 
+@router.get(
+    Paths.relationships,
+    summary="List all relationships",
```

Comment:
Do you have a user case in mind where this would be useful? There are e.g. 15k concepts in each years Combined Nomenclature; assuming we have 2023, 2024, and 2025, that is 50k relationships, most of which don't seem like they would answer a specific question.

I recognize that we need some additional endpoints, but don't see where this one would fit in. But maybe I am just missing something obvious, as usual!

Reply:
Nope sorry, this shouldn't have made it to this PR :)